cubedash/summary/_stores.py

```diff
@@ -1643,6 +1643,28 @@ def find_datasets_for_region(
             offset=offset,
         )
 
+    def find_products_for_region(
+        self,
+        region_code: str,
+        year: int,
+        month: int,
+        day: int,
+        limit: int,
+        offset: int = 0,
+    ) -> Iterable[Dataset]:
```

Comment:
Doesn't return `Iterable[Dataset]` at the moment (it's returning strings)... maybe `Iterable[DatasetType]` if we intend it to "find products"? (see other comment)